backend/spec/features/admin/configuration/general_settings_spec.rb

```diff
@@ -39,5 +39,31 @@
 
       expect(page).to have_content(Spree.t(:clear_cache_ok))
     end
+
+    context "with Spree mounted to a custom path" do
```

Comment:
I don't think we need to add another spec just for this on the already slow backend.  The hardcoded url should have been caught during code review.